src/openforms/prefill/attributes_generator.py

```diff
@@ -0,0 +1,122 @@
+from dataclasses import dataclass
+
+from openapi_parser import parse
+
+from openforms.template import render_from_string
+
+
+class AttributeGeneratorException(Exception):
+    pass
+
+
+FILE_TEMPLATE = """
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+class Attributes(models.TextChoices):
+    \"\"\"
+    This code was (at some point) generated from the management command below. Names and labels are in Dutch if the spec was Dutch
+    specs: {{ url }}
+    schema: {{ schema }}
+    command: {{ command }}
+    \"\"\"
+
+    {% for attribute in attributes %}{{ attribute.attribute_variable_name }} = "{{ attribute.attribute_value }}", _("{{ attribute.attribute_label }}")
+    {% endfor %}
+"""
+
+
+@dataclass
+class OpenApi3AttributesGenerator:
+    url: str = ""
+    schema: str = ""
+    command: str = ""
+    # TODO features present in the generate_prefill_from_spec command
+    # - Specify a service and then generate the attributes from the OAS of this service
+    # - Specify a path and then only generate the attributes for this path
+    # - If no path/schema are specified, print all paths and schemas in the OAS
+    # - Option to add '_embedded.' in paths for $refs
+    # Additional features that could be good
+    # - Keep track of the titles in addition to the properties names in the schema
+
+    def get_specifications(self):
+        if self.url:
+            # The content types supported in the openapi3-parser are these:
+            # https://github.com/manchenkoff/openapi3-parser/tree/master/src/openapi_parser/home/silvia/repositories/openapi3-parser/src/openapi_parser/enumeration.py
+            # However, in the haal centraal OAS we find content types such as "application/hal+json". So with
+            # strict_enum=False, the parser just checks that the content type is a string
+            return parse(uri=self.url, strict_enum=False)
+
+        raise AttributeGeneratorException(
+            "No URL to the OAS specifications was provided."
+        )
+
+    def get_attributes(self, property_schema):
```

Comment:
```suggestion
    def get_attributes(self, property_schema):
```
I'd appreciate type hints to make it easier to reason about what `property_schema` is exactly :grimacing: